### PR TITLE
bump k8s libs

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/2844.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -29,7 +30,7 @@
   version = "3.1.10"
 
 [[constraint]]
-  branch = "master"
+  branch = "bump-k8s-libs"
```
Comment (xh3b4sd) [inline Gopkg.toml]: Waiting for https://github.com/giantswarm/apiextensions/pull/63.

File: Gopkg.toml [hunk 2]
```diff
@@ -53,7 +54,7 @@
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "bump-k8s-libs"
```
Comment (xh3b4sd) [inline Gopkg.toml]: Waiting for https://github.com/giantswarm/operatorkit/pull/176.

**Comment by xh3b4sd** [comment]
Successfully tested on geckon.

**Review by rossf7 — APPROVED** [review]
LGTM once the related PRs are merged.

**Review by kopiczko — APPROVED** [review]

**Review by fgimenez — APPROVED** [review]
LGTM

**Pull request merged**